<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="passportView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>911</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>930</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Change data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>930</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>930</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save scan</string>
    </property>
   </widget>
   <widget class="QLabel" name="passportImg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>911</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>990</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1283</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
